Sources/NIOPosix/SelectorWSAPoll.swift

```diff
@@ -0,0 +1,178 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the SwiftNIO open source project
+//
+// Copyright (c) 2025 Apple Inc. and the SwiftNIO project authors
+// Licensed under Apache License v2.0
+//
+// See LICENSE.txt for license information
+// See CONTRIBUTORS.txt for the list of SwiftNIO project authors
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+//===----------------------------------------------------------------------===//
+
+#if os(Windows)
+import CNIOWindows
+import NIOCore
+import WinSDK
+
+extension SelectorEventSet {
+    var wsaPollEventSet: Int16 {
+        var result: Int16 = 0
+        if self.contains(.read) || self.contains(.readEOF) {
```

Comment:
We shouldn't map `readEOF` to `POLLRDNORM`: we'll almost never unset it, even when we don't want to read. WSAPoll can't send us the necessary signal so let's never handle it.